apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -54,6 +54,7 @@
  * Definitions
  ****************************************************************************/
 #define BUFLEN 64
+#define BUF_MAX	20
```

Comment:
please align here.

Reply:
Also need rename to understand more rename.
below is an example
#STDIO_BUFLEN 64
#VFS_CONTENTS_LEN 20
.
.

Reply:
done